package-lock.json

```diff
@@ -217,6 +217,12 @@
       "integrity": "sha512-dInnr7nSPsofnLggOf70xvsInUjf3tRrE8XmxsioXALWQHkwEWi7RhTBCa9mYmiUDHMtuanSDN/JOW187ChIhw==",
       "dev": true
     },
+    "@types/eslint-visitor-keys": {
```

Comment:
Interesting. @sophiewu6 are these changes the same as the ones you had?

Reply:
Yes. They look the same.

Reply:
Hi @seanlip & @sophiewu6,
I spent good time debugging it but I didn't much of it.
Do you know why this is happening? Also, i'll provide a testing doc

Reply:
I'm afraid I don't know. @sophiewu6 did you follow up with the npm folks? If so could you please post a link to the issue thread where the discussion is happening? If not I would suggest following up as this seems to be an npm issue.

Reply:
https://npm.community/t/npm-errors-blocked-my-workflow/8213 Here is the link. I'll post this in the issue as well.